.kontinuous/values.yaml

```diff
@@ -0,0 +1,28 @@
+global:
+  registry: harbor.fabrique.social.gouv.fr
```

Comment:
```suggestion
```

Reply:
c'est par défaut, et sinon il vaut mieux le mettre dans la config que dans `global` (dans lesquelles ça se retrouva aussi in fine)
ex: https://github.com/SocialGouv/cdtn-api/blob/80362acab6c65e8e74a6505af635c224f9c374ec/.kontinuous/config.yaml#L6